code/game/objects/items/weapons/vacuumcleaner.dm

```diff
@@ -0,0 +1,184 @@
+#define CHANNEL_VACUUMCLEANER 764 //temporary - do not merge this
+								  // see below comments in switchOn proc
+/obj/item/weapon/vacuumcleaner
+	desc = "There is a slot for a power cell and a place to tie a trash bag."
+	name = "vacuum cleaner"
+	icon = 'icons/obj/janitor.dmi'
+	icon_state = "vacuum_cleaner"
+	hitsound = "sound/weapons/whip.ogg"
```

Comment:
shouldnt this be more of a toolbox crunch or are you hitting with the cord

Reply:
Make the hit sound the tf2 pan sound. It's in the sounds already